langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -188,6 +308,22 @@ public isolated function removeIfHasKey(table<MapType> key<KeyType> t, KeyType k
 #
 # This panics if any member cannot be removed.
 #
+# ```ballerina
+# table<record {|string name;|}> students = table [
+#     {name: "Jo"},
+#     {name: "Sam"}
+# ];
+# 
+# students.removeAll() is () ⇒ true
+#
+# table<record {|string name;|}> readonlyStudents = <readonly> table [
+#     {name: "Jo"},
+#     {name: "Sam"}
+# ];
```

Comment:
How about having a simpler name for this table instead of `readonlyStudents`? We can have something like 
```suggestion
# table<record {|int score;|}> scores = <readonly> table [
#     {score: 30},
#     {score: 40}
# ];
```